### PR TITLE
docs: Add HugeNoob profile picture and PPP

### LINKED ISSUE #24: Update AboutUs and add skeletal PPP - HugeNoob
- [x] Update the AboutUs page by adding details and a photo.
- [x] Create a skeletal version of your Project Portfolio Page (PPP). 

https://nus-cs2103-ay2324s1.github.io/website/schedule/week7/project.html#tp-get-ready-for-iterations

### PR DESCRIPTION
**Pull request by HugeNoob**
## What type of PR is this? (check all applicable)

-   [ ] Refactor
-   [ ] Feature
-   [ ] Bug Fix
-   [ ] Optimization
-   [x] Documentation Update

## Description

1. Add my profile picture
2. Add my PPP
3. Update the AboutUs page to include my details

Please help to check that I've added them into the correct folders. Thanks!

## Related Tickets & Documents

Closes #24 

## Added/updated tests?

_We encourage you to keep the code coverage percentage at 75% and above for new changes._

-   [x] Yes: Here's a [deployment on a separate branch](https://docs-hugenoob-website-docs.coordimate.pages.dev/).
-   [ ] No, and this is why: 
-   [ ] No, there are UI changes that I'm unable to cover.
-   [ ] I need help with writing tests

### CODE REVIEW

**Review by pra-navi — APPROVED** [review]
LGTM

**Review by jiakai-17 — CHANGES_REQUESTED** [review]
LGTM, but according to the [2103T website](https://nus-cs2103-ay2324s1.github.io/website/schedule/week7/project.html), the various headings still need to be included, though placeholders can be used for the content.

Perhaps you could add back the headers?

File: docs/team/hugenoob.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+CoordiMate is a **desktop app for event planners**. It helps you **manage your contacts and tasks** for your events, so that you can focus on the event itself.
+
+CoordiMate is **optimized for use via a Command Line Interface (CLI)** while still having the benefits of a Graphical User Interface (GUI).
+
+If you can type fast, CoordiMate can help you get your contact management tasks done **faster than traditional GUI apps**.
+
+Given below are my contributions to the project.
+
+To be added soon. Stay tuned!
```
[the remark this answers is not in the document]
Comment (HugeNoob) [reply docs/team/hugenoob.md]: Done, [check it out](https://docs-hugenoob-website-docs.coordimate.pages.dev/team/hugenoob).

[the remark this answers is not in the document]
Comment (jiakai-17) [reply docs/team/hugenoob.md]: LGTM, thanks for the changes.

**Review by jiakai-17 — APPROVED** [review]

**Review by pra-navi — DISMISSED** [review]

**Comment by pra-navi** [comment]
LGTM, but should we standardise the format of team/name.md to follow the example given under this module (essentially jiakai's version)

**Comment by HugeNoob** [comment]
> LGTM, but should we standardise the format of team/name.md to follow the example given under this module (essentially jiakai's version)

The format to follow is specified in the [CS2103T website](https://nus-cs2103-ay2324s1.github.io/website/schedule/week7/project.html#ppp-format-2).
image

**Review by pra-navi — APPROVED** [review]
LGTM!

**Pull request merged**